packages/core/src/actions/send.ts

```diff
@@ -124,6 +124,7 @@ function resolveSendTo(
     snapshot,
     {
       to: targetActorRef,
+      actorId: typeof resolvedTarget === 'string' ? resolvedTarget : undefined,
```

Comment:
we could name this `targetId` to slightly disambiguate it (with `actorId` it's not immediately obvious if it contains the source's id or target's id)